aten/src/ATen/native/native_functions.yaml

```diff
@@ -601,6 +601,9 @@
 - func: affine_grid_generator_backward(Tensor grad, int[] size, bool align_corners) -> Tensor
   variants: function
 
+- func: _is_true(Tensor self) -> bool
```

Comment:
nit: Naming: in general, for out-of-place operations, `vmap(f)(xs)` should be equivalent to `torch.stack([f(x) for x in xs])`.

`_is_true` behaves differently from that: instead of returning one bool per value in xs, we are returning a single boolean.

In JAX the operations that have this property are named specially and prefixed with a `p`, e.g., psum, pmean, etc. Those are also a bit trickier to use (they require an axis name), but that's a different story.

It would be nice for the name of this to reflect that it is doing something special w.r.t. to vmap, but I don't have a good idea for a name. Since this op is supposed to be "private" (prefixed with an underscore) we could just punt the naming scheme to later.